### DISCUSSION TITLE
Blacklist/exclude leds from processing

### DISCUSSION THREAD

**Discussion by timota** [awawa-dev/HyperHDR#675] [Q&A] [answered]
Hi guys. 

Need your help. 

I have a led strip mounted in niche around my TV. As an example(dont pay attention to numbers, please):
Left: 120 leds
Right: 120 Leds
Top: 200 Leds
Bottom: 200 Leds

To mach leds with a tv I need to exclude some leds from processing, simply turn them off, eg: 
Left: 
 - exclude 0-20 
 - include 21-100
 - exclude 101-120

Same applicable to other sides

Is possible to do with a hyperhdr ? Or do i need to cut strip and treat each side as separate led string and use gaps ?

Thank you

**Comment by awawa-dev** [comment, marked as answer]
Hi
With HyperHDR you can use both solutions: without cutting you can turn some LEDs off permanently  ('disable' option in LED strip designer https://github.com/awawa-dev/HyperHDR/pull/379) or you can cut the strip that will still be connected and remove the missing LEDs ('delete' option in LED strip designer https://github.com/awawa-dev/HyperHDR/pull/379).

**Reply by timota** [reply]
Did not know that that possible. Thank you. Will try.
